src/test/run-make-fulldeps/coverage-reports/expected_show_coverage.closure.txt

```diff
@@ -125,36 +126,397 @@
+  135|      1|    let short_used_not_covered_closure_macro = | used_arg: u8 | println!("not called");
+  136|      1|    let _short_unused_closure_macro = | _unused_arg: u8 | println!("not called");
+  137|       |    // The closure assignment above is executed, with a line count of `1`, but the `println!()`
+  138|       |    // could not have been called, in the `not_covered` cases, but there is no indication it wasn't.
+  139|       |
+  140|       |    // ...but adding block braces gives the expected result, showing the block was not executed.
   141|      1|    let _short_unused_closure_block = | _unused_arg: u8 | { println!("not called") };
                                                                         ^0
-  142|      1|
+  142|       |
   143|      1|    let _shortish_unused_closure = | _unused_arg: u8 | {
   144|      0|        println!("not called")
-  145|      1|    };
-  146|      1|
+  145|      0|    };
+  146|       |
   147|      1|    let _as_short_unused_closure = |
   148|       |        _unused_arg: u8
-  149|      1|    | { println!("not called") };
-                    ^0
-  150|      1|
+  149|      0|    | { println!("not called") };
+  150|       |
   151|      1|    let _almost_as_short_unused_closure = |
   152|       |        _unused_arg: u8
-  153|      1|    | { println!("not called") }
-                    ^0
-  154|      1|    ;
-  155|      1|}
+  153|      0|    | { println!("not called") }
+  154|       |    ;
+  155|       |
+  156|       |    // Same example as `_short_unused_closure` above, and even broken across
+  157|       |    // multiple lines, the `println!(...)` should show `0` executions, but it
+  158|       |    // shows `1`. This should be fixable, now that most other macros have more
+  159|       |    // intuitive coverage results.
+  160|      1|    let _short_unused_closure_line_break_no_block = | _unused_arg: u8 |
+  161|       |println!("not called")
+  162|       |    ;
+  163|       |
+  164|      1|    let _short_unused_closure_line_break_no_block2 =
+  165|       |        | _unused_arg: u8 |
+  166|       |            println!(
+  167|       |                "not called"
+  168|       |            )
+  169|       |    ;
+  170|       |
+  171|      1|    let short_used_not_covered_closure_line_break_no_block_embedded_branch =
+  172|       |        | _unused_arg: u8 |
+  173|       |            println!(
+  174|       |                "not called: {}",
+  175|       |                if is_true { "check" } else { "me" }
+  176|       |            )
+  177|       |    ;
+  178|      1|    if is_false {
+  179|      0|        short_used_not_covered_closure_macro(0);
+  180|      0|        short_used_not_covered_closure_line_break_no_block_embedded_branch(0);
+  181|      1|    }
+  182|      1|    short_used_covered_closure_macro(0);
+  183|      1|}
+
+/usr/local/google/home/richkadel/rust/library/std/src/macros.rs:
```

Comment:
This file path shouldn't be absolute

Reply:
This issue (absolute path in coverage report) is resolved.

I reverted the changes to the `closure.rs` test, which avoids generating coverage results with the coverage in `println!()`.

The changes I made (now reverted) were incorporated in more extensive testing of the PR that resolves the tested issue (#84897)